lib/Logging/LogRepository.php

```diff
@@ -34,13 +34,13 @@ public function getLogs(
 
     if ($dateFrom instanceof \DateTimeInterface) {
       $query
-        ->andWhere('l.createdAt > :dateFrom')
-        ->setParameter('dateFrom', $dateFrom->format('Y-m-d H:i:s'));
+        ->andWhere('l.createdAt >= :dateFrom')
+        ->setParameter('dateFrom', $dateFrom->format('Y-m-d H:i:s 00:00:00'));
```

Comment:
I think this is a typo `Y-m-d H:i:s 00:00:00` should probably be `Y-m-d 00:00:00`.